### PR TITLE
Bdog 651

### PR DESCRIPTION
**Pull request by bennetimo**

### CODE REVIEW

File: build.sbt [hunk 1]
```diff
@@ -27,24 +27,25 @@ lazy val root = Project(libName, file("."))
...
   )
   .settings(
-    scalaVersion := "2.11.12",
-    crossScalaVersions := Seq("2.11.12", "2.12.8"),
+    scalaVersion := "2.12.10",
+    crossScalaVersions := Seq("2.11.12", "2.12.10"),
     libraryDependencies ++= Seq(
-      "org.apache.commons" % "commons-lang3" % "3.4",
+      "org.apache.commons" % "commons-lang3" % "3.9",
       // force dependencies due to security flaws found in jackson-databind < 2.9.x using XRay
```
Comment (cjwebb) [inline build.sbt]: Is this comment now obsolete?

File: build.sbt [hunk 2]
```diff
@@ -27,24 +27,25 @@ lazy val root = Project(libName, file("."))
...
-      "com.fasterxml.jackson.core"     % "jackson-core"            % "2.9.7",
-      "com.fasterxml.jackson.core"     % "jackson-databind"        % "2.9.7",
-      "com.fasterxml.jackson.core"     % "jackson-annotations"     % "2.9.7",
-      "com.fasterxml.jackson.datatype" % "jackson-datatype-jdk8"   % "2.9.7",
-      "com.fasterxml.jackson.datatype" % "jackson-datatype-jsr310" % "2.9.7",
-      "commons-io"                     % "commons-io"              % "2.4",
+      "com.fasterxml.jackson.core"     % "jackson-core"            % "2.10.3",
```
Comment (christopherjturner) [inline build.sbt]: do we still need to be pinning jackson? It was done to mitigate the deserialization vuln ~ 2 years back

File: build.sbt (hunk 1 above)
Comment (bennetimo) [reply build.sbt]: Whoops, yep removed

File: build.sbt (hunk 2 above)
Comment (bennetimo) [reply build.sbt]: cool, simplified the dependencies

**Pull request merged**